Seedshop/Tooltip.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using Godot;
+
+public partial class Tooltip : Popup
+{
+	public string origin = "";
+	public string slot = "";
+	public bool valid = false;
+
+	public override void _Ready()
+	{
+		if (origin == "Seedshop" && slot != "")
+		{
+			GetNode<Label>("ColorRect/MarginContainer/VBoxContainer/Label").Text = slot;
+			GD.Print(GetNode<Label>("ColorRect/MarginContainer/VBoxContainer/Label").Text);
+			valid = true;
+		}
+		if (valid)
```

Comment:
this `if` does nothing

Reply:
Was supposedto be before but still does nothing -> removed